<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keyword Search</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>1131</width>
      <height>751</height>
     </rect>
    </property>
    <property name="title">
     <string>Key word Search</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>441</width>
       <height>621</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="keyword_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>431</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Keyword Name </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Medicine Got From database</string>
      </property>
     </widget>
     <widget class="QLabel" name="keyw_databse">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>431</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Keyword Name </string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>260</y>
        <width>461</width>
        <height>391</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>static files/placeholder_product.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Your Entered Keyword</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="backtodash">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>30</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Back to Dashboard</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>130</y>
       <width>681</width>
       <height>621</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Description</string>
      </attribute>
      <widget class="QTextBrowser" name="desc_med">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>681</width>
         <height>591</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Side Effects</string>
      </attribute>
      <widget class="QTextBrowser" name="effect_med">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>681</width>
         <height>591</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Uses</string>
      </attribute>
      <widget class="QTextBrowser" name="uses_med">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>681</width>
         <height>591</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Dosage</string>
      </attribute>
      <widget class="QTextBrowser" name="dosage_med">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>681</width>
         <height>591</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Substitutes</string>
      </attribute>
      <widget class="QTextBrowser" name="subs_med">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>681</width>
         <height>591</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
